src/instructlab/lab.py

```diff
@@ -38,6 +38,10 @@
 logging.getLogger("openai").setLevel(logging.ERROR)
 logging.getLogger("httpx").setLevel(logging.ERROR)
 
+# Intel Gaudi: workaround for race condition in oneMKL [HS-1795]
```

Comment:
Is there anything tracking this externally? When/how do we know this can be removed?

Reply:
The problem is tracked in the internal ticket `HS-1795` in a private Jira instance. Only Intel engineers and a few RH engineers have access to that Jira instance.